README.md

```diff
@@ -53,6 +53,28 @@ You might optionally want to run one worker towards the `failed` queue.
 Pretty much normal Python + WSGI setup, just install the requirements and serve using WSGI. For Ubuntu 14.04,
 an Ansible role is provided.
 
+#### Init
+
+There are example systemd service files in the 'extra' directory. The examples
+use a specific user and utilize gunicorn. They have been tested on CentOS 7.
+
+To use, modify as needed (user, group, and path), copy to `/etc/systemd/service`
```

Comment:
I guess it would be nice to have working defaults that work by just copying the files out, which of course means some assumptions on paths should be mentioned, like in these example maybe paths could be `/home/socialrelay/social-relay` for working directory and `/home/socialrelay/social-relay/venv` for location of virtualenv? Then if someone wants to they can make sure their install fits these assumptions and can enjoy ready working scripts.

Otherwise looks good!